services/harmony/app/util/variables.ts

```diff
@@ -99,6 +99,37 @@ export function getCoordinateVariables(variables: CmrUmmVariable[]): CmrUmmVaria
   return variables.filter((v) => v.umm.VariableType === coordinateType);
 }
 
+/**
+ * validate variable and queryVars parameters, throw exception if failed
+ * @param variableIds - The variable ids in url path
+ * @param queryVars - The variables in query params
+ */
+function validateVariables(variableIds: string[], queryVars: string | string[]): void {
```

Comment:
Add a unit test for this if you haven't already.

Reply:
I was just refactoring existing code. No good deed goes unpunished. :)